pkg/networkserver/downlink.go

```diff
@@ -423,12 +406,19 @@ func (ns *NetworkServer) generateDownlink(ctx context.Context, dev *ttnpb.EndDev
 	default:
 		return nil, genState, errNoDownlink
 	}
+	if mType == ttnpb.MType_UNCONFIRMED_DOWN && len(dev.MACState.PendingRequests) > 0 &&
+		(dev.MACState.DeviceClass == ttnpb.CLASS_B ||
```

Comment:
Do we always want an immediate answer in class B, also in 1.1+?

Reply:
AFAIK no LoRaWAN spec version released so far contains anything similar to the clause in 1.1 class C specifying the timeout between confirmed downlink transmissions, so I'd say yes.